components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -108,35 +114,177 @@ export default {
 		iconType: "ui",
 		iconName: "ChevronDown",
-		position: "right"
+		position: "right",
 	},
 };
 
+// Story specific templates
+const SizingTemplate = (args) => html`
+	<div
+		style=${styleMap({
+			display: "flex",
+			gap: "24px",
+			flexWrap: "wrap",
+		})}
+	>
+		${["s", "m", "l", "xl"].map(
+			(size) => html`
+				<div
+					style=${styleMap({
+						display: "flex",
+						gap: "16px",
+						flexDirection: "column",
+						alignItems: "center",
+						flexBasis: "80px",
+					})}
+				>
+					${Typography({
+						semantics: "detail",
+						size: "s",
+						content: [
+							{
+								s: "Small",
+								m: "Medium",
+								l: "Large",
+								xl: "Extra-large",
+							}[size],
+						],
+						customStyles: {
+							"white-space": "nowrap",
+							"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+						},
+					})}
+					${Template({ ...args, size })}
+				</div>
+			`,
+		)}
+	</div>
+`;
+
+const CustomIconTemplate = (args) => html`
+	<div
+		style=${styleMap({
+			display: "flex",
+			gap: "24px",
+			flexWrap: "wrap",
+		})}
+	>
+		<div
+			style=${styleMap({
+				display: "flex",
+				gap: "16px",
+				flexDirection: "column",
+				alignItems: "center",
+				flexBasis: "80px",
+			})}
+		>
+			${Typography({
+				semantics: "detail",
+				size: "s",
+				content: ["UI icon"],
+				customStyles: {
+					"white-space": "nowrap",
+					"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+				},
+			})}
+			${Template({
+				...args,
+				iconName: "ArrowDown100",
+				iconType: "ui",
+			})}
+		</div>
+		<div
+			style=${styleMap({
+				display: "flex",
+				gap: "16px",
+				flexDirection: "column",
+				alignItems: "center",
+				flexBasis: "80px",
+			})}
+		>
+			${Typography({
+				semantics: "detail",
+				size: "s",
+				content: ["Workflow icon"],
+				customStyles: {
+					"white-space": "nowrap",
+					"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+				},
+			})}
+			${Template({
+				...args,
+				iconName: "Add",
+				iconType: "workflow",
+			})}
+		</div>
+	</div>
+`;
+
+// Stories
 export const Default = Template.bind({});
 Default.args = {};
 
 export const WithLabel = Template.bind({});
+WithLabel.storyName = "With label";
 WithLabel.args = {
 	label: "Select",
 };
 
 export const Disabled = Template.bind({});
 Disabled.tags = ["vrt-only"];
 Disabled.args = {
-	isDisabled: true
+	isDisabled: true,
 };
 
 export const Quiet = Template.bind({});
+Quiet.storyName = "Quiet with label";
 Quiet.tags = ["vrt-only"];
 Quiet.args = {
-	isQuiet: true
+	isQuiet: true,
+	label: "All",
 };
 
 export const Express = Template.bind({});
-Express.tags = ["vrt-only"];
+Express.tags = ["vrt-only", "!autodocs"];
 Express.args = {
-	express: true
+	express: true,
+};
+
+export const Sizing = SizingTemplate.bind({});
+Sizing.tags = ["docs-only"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const Open = Template.bind({});
+Open.tags = ["docs-only"];
```

Comment:
What's the distinction between using `vrt-only` and `docs-only` tags? They both remove the story from the sidebar, but our `docs-only` tagged stories have the chromatic snapshot disabled. How should I be deciding which one to use? We don't technically need the `vrt-only` tag if we want it to be included in the chromatic snapshot, right? 

I'll have to look this up!

Reply:
The tags are now changed to use the newer built-in Chromatic tags though (per Cassondra's suggestion).